strings/french.strings

```diff
@@ -138,6 +138,27 @@
 /* controller/actions/identity.js */
 "Invalid username or password" = "Nom d'utilisateur ou mot de passe invalide";
 
+/* controller/actions/chatroom.js */
+"Failed to create chat channel.": "Échec de la création du canal de discussion.",
+
+/* controller/actions/chatroom.js */
+"Failed to join chat channel.": "Échec de la connexion au canal de chat.",
+
+/* controller/actions/chatroom.js */
+"Failed to rename chat channel.": "Impossible de renommer le canal de chat.",
+
+/* controller/actions/chatroom.js */
+"Failed to change chat channel description.": "Échec de la modification de la description du canal de chat.",
+
+/* controller/actions/chatroom.js */
+"Failed to leave chat channel.": "Impossible de quitter le canal de chat.",
+
+/* controller/actions/chatroom.js, controller/actions/group.js */
+"Failed to delete chat channel.": "Échec de la suppression du canal de chat.",
+
+/* controller/actions/chatroom.js */
+"Failed to add message.": "Échec de l'ajout du message.",
+
```

Comment:
Same same

Reply:
@pieer, what does this `Same same` mean?

Reply:
Same comment as above

Reply:
I don't get what you mean? Is the translation incorrect?